lib/runner.js

```diff
@@ -529,7 +533,13 @@ Runner.prototype.runTests = function (suite, fn) {
     }
 
     if (test.isPending()) {
-      self.emit('pending', test);
+      if (self.forbidPending) {
+        test.isPending = alwaysFalse;
```

Comment:
Additionally, Coveralls indicates that the `fail` function's pending test ignore branch is never hit: https://coveralls.io/builds/11947829/source?filename=lib%2Frunner.js#L224 However, to confirm that we aren't just missing a test for this case, I'd have to go look at whether we have tests for all... five? six?... cases where tests can be pending: `describe.skip`, `it.skip`, `it("title and not callback")`, `this.skip` in `it`, `this.skip` in `beforeEach`, `this.skip` in `before`...